omnigibson/robots/untucked_arm_pose_robot.py

```diff
@@ -0,0 +1,146 @@
+from abc import abstractmethod
+
+import torch as th
+
+from omnigibson.robots.mobile_manipulation_robot import MobileManipulationRobot
+from omnigibson.utils.python_utils import assert_valid_key, classproperty
+
+
+class UntuckedArmPoseRobot(MobileManipulationRobot):
```

Comment:
Nit: Should these actually be two separate classes? Is there any case of a mobile manipulator that doesn't have a un / tucked position?

Reply:
Hmm what do you mean? `UntuckedArmPoseRobot` handles a fixed set of untucked arm poses (e.g. diagonal15, vertical, etc). `MobileManipulationRobot` handles the general logic of untucked/tucked position.